modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/Tabs.js

```diff
@@ -44,6 +29,26 @@ export default function ({
 	navSpecificationsId,
 }) {
 	const [activeTabKeyValue, setActiveTabKeyValue] = useState(0);
+	const [firstRender, setFirstRender] = useState(true);
+
+	const display = (tabId) => {
+		for (let i = 0; i < tabs.length; i++) {
```

Comment:
We would prefer `tabs.forEach(tab => {...` here

Reply:
yep totally